mne/transforms.py

```diff
@@ -432,7 +432,14 @@ def _ensure_trans(trans, fro='mri', to='head'):
 
 def _get_trans(trans, fro='mri', to='head'):
     """Get mri_head_t (from=mri, to=head) from mri filename."""
-    if isinstance(trans, str):
+    try:
+        _validate_type(trans, 'path-like', 'trans')
+        trans = str(trans)
+        path_like = True
+    except TypeError:
+        path_like = False
```

Comment:
can you rather create a private function next to _validate_type.
I would call it _check_file_like and it would return True or False if is file-like
thx

Reply:
@agramfort Is 195b845 to your liking? I've named the function `_check_path_like` (i.e., not `_check_file_like()`, as you had proposed) to avoid confusion between paths and file objects.
Can certainly add tests too.